docker-compose.yml

```diff
@@ -16,7 +16,8 @@ services:
 
   # Hasura Postgres -> GraphQL engine
   hasura:
-    image: hasura/graphql-engine:v2.0.10
+    platform: linux/arm64
+    image: hasura/graphql-engine:latest
```

Comment:
I would be careful to change the Hasura version, since updates can change the exposed API structure. This also needs to match the version in the rcos-data repo.

Reply:
Ditto